service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -135,4 +136,11 @@ private void run1210(CaseDetails caseDetails) {
             "EDIT_HEARING", migrationId);
         caseDetailsData.put("hearingOption", HearingOptions.EDIT_PAST_HEARING);
     }
+
+    private void run1270(CaseDetails caseDetails) {
+        String migrationId = "DFPL-1270";
+        migrateCaseService.doCaseIdCheck(caseDetails.getId(), 1656581935993992L, migrationId);
+        caseDetails.getData().putAll(migrateCaseService.removeCaseSummaryByHearingId(getCaseData(caseDetails),
+            migrationId, UUID.fromString("9e6bc008-3577-4c6b-8a29-521c9027b17f")));
```

Comment:
I imagine it'll be fine but perhaps worth confirming that the UUID on the ticket is the right one - just had a worry with the function name being `removeCaseSummaryByHearingId`

Reply:
Looks like the case summary using hearing id as an identifier. I will double check it.

Reply:
I have checked that it seems the relation between hearing and case summary is 1 to 1. That means there is only a single case summary for each hearing.

Reply:
and verified that the id in case summary is actually using the corresponding hearing id.